tests/test_server_config.py

```diff
@@ -1353,15 +1353,26 @@ async def test_server_proto_configure_error(self):
 
             err = {
                 'type': 'SchemaError',
-                'message': 'danger',
+                'message': 'danger 1',
                 'context': {'start': 42},
             }
+
+            # N.B: When CONFIGURE CURRENT DATABASE is run, the new
+            # setting becomes live immediately on the current
+            # connection, but does not become live on other
+            # connections until after a scheduled background
+            # reintrospection is done.
+            #
+            # There is a bug related to this (#7330) in which an
+            # *older* scheduled reintrospection can temporarily
```

Comment:
Ugh, why introspection lock (#7485) didn't fix #7330 again? Cache-related issues?

Reply:
The introspection lock makes sure the introspections don't get reordered, but doesn't prevent an older introspection from overwriting a newer "session level" application of a database config.